tests/webapp/api/test_performance_data_api.py

```diff
@@ -0,0 +1,115 @@
+import datetime
+import time
+
+import pytest
+from django.core.urlresolvers import reverse
+from rest_framework.test import APIClient
+
+from treeherder.model.models import (MachinePlatform,
+                                     Option,
+                                     OptionCollection)
+from treeherder.perf.models import (PerformanceDatum,
+                                    PerformanceFramework,
+                                    PerformanceSignature)
+
+
+@pytest.fixture
+def summary_perf_signature(test_repository, test_perf_signature):
+
+    framework = PerformanceFramework.objects.get(
+        name='test_talos')
+    option = Option.objects.create(name='opt')
+    option_collection = OptionCollection.objects.create(
+        option_collection_hash='my_option_hash',
+        option=option)
+    platform = MachinePlatform.objects.create(
+        os_name='win',
+        platform='win7',
+        architecture='x86',
+        active_status='active')
+
```

Comment:
you don't want to recreate these. Just get() the existing ones when you're constructing the extra performance signature below.